### PR TITLE
Fix code comments BE

### PR DESCRIPTION
**Pull request by AndreiTocu**

### CODE REVIEW

File: codewellBe/t/Controller/comment.t [hunk 1]
```diff
@@ -197,8 +197,15 @@ subtest 'Code comments' => sub {
     });
 
   # Check the comments for a logged in user and a valid code_id
-  my $comment = create_comment($t, 'First comment', $code_id);
-  my $second_comment = create_comment($t, 'Second comment', $code_id);
+  my %comment_request = (
+    body => 'First comment',
+    code_id => $code_id,
+    parent_comment_id => undef,
```
Comment (andreisamoila74) [inline codewellBe/t/Controller/comment.t:203]: you don't need to include these fields here

Comment (andreisamoila74) [reply codewellBe/t/Controller/comment.t:203]: they would be added by default with undef value in the db

File: codewellBe/t/Controller/comment.t [hunk 2]
```diff
@@ -212,12 +219,16 @@ subtest 'Code comments' => sub {
...
     },
     {
       user_id =>  $second_comment->user_id,
       date_posted => $second_comment->date_posted,
-      body => $second_comment->body
+      body => $second_comment->body,
+      parent_comment_id => undef,
+      line_number => undef
     }
   );
```
Comment (andreisamoila74) [inline codewellBe/t/Controller/comment.t:234]: also test some comments with defined values for the new fields

File: codewellBe/t/lib/TestHelper.pm [hunk 3]
```diff
@@ -31,12 +31,14 @@ sub create_code {
 }
 
 sub create_comment {
-  my ($t, $body, $code_id) = @_;
+  my ($t, %comment_request) = @_;
   $t->post_ok('/api/comment/add' => json => {
```
Comment (andreisamoila74) [inline codewellBe/t/lib/TestHelper.pm:35]: json => %comment_request

**Review by andreisamoila74 — CHANGES_REQUESTED** [review]

**Review by andreisamoila74 — CHANGES_REQUESTED** [review]
the pr is good, but please change the name of the commit in 'Fix code comments BE'

**Review by S7012MY — APPROVED** [review]

**Review by andreisamoila74 — APPROVED** [review]

**Pull request merged**